.gitignore

```diff
@@ -1,2 +1,27 @@
-target/
-.idea
```

Comment:
You removed these lines from the .gitignore. Please add them back.

Reply:
got them back